### ISSUE TITLE
result of post processing

### ISSUE THREAD

**Issue by fyangneil** [antonilo/unsupervised_detection#11]
i use the provided model and post processing code on davis16 dataset, the final result is 69.3 which is lower than 71.5 reported in paper. I am not sure if the result is correct.

**Comment by antonilo** [comment]
I think you might have an issue with image sizes. Are you sure you are running the post-processing on the final resolution (640x480)? Did you use our provided script to compute the metric or did you implement your own?

**Comment by fyangneil** [comment]
@antonilo I tried again and still got 69.3 on DAVIS
![Screenshot 2020-10-24 225146](https://user-images.githubusercontent.com/46222158/97098134-f09d4c00-164f-11eb-9f00-cb5162a12476.png)
The output mask is 480*854 not 640*480. I did use your provided model and post-processing code and did not change anything.

**Comment by antonilo** [comment]
Do you get the same results as in here http://rpg.ifi.uzh.ch/data/detection_results.zip> Did you use this [evaluation script](https://github.com/antonilo/unsupervised_detection/blob/master/test_generator.py)? If not, I might need more details to debug the problem (TensorFlow version used, for example)

**Comment by fyangneil** [comment]
I did use your provided evaluation script. but I do not think I get the exactly same detection results as yours, because based on your detection results and evaluation tool, I can get the number reported in paper.
________________________________

**Comment by antonilo** [comment]
Might be that you used a sub-optimal checkpoint. How many epochs did you train?

**Comment by fyangneil** [comment]
I use the model you provided in github

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by WeidiXie** [comment]
@fyangneil Hi, anything follows this ? have you reproduced the same number ? eg. 71 ?

**Comment by fyangneil** [comment]
no, I cannot reproduce the number reported in paper
________________________________

**Comment by antonilo** [comment]
Sorry, but it is very difficult to help without details. I would say that the problem is in the way images are pre-processed (reshaping and central cutting). The changes might have affected the ckpts. What if you train a new ckpt (from scratch or resuming from the one available)? The predicted masks have been produced with the model available online.

**Comment by fyangneil** [comment]
But, I did not change anything. I simply used the model online and run the test script for DAVIS dataset, the result is lower than the number reported in paper.
________________________________

**Comment by fyangneil** [comment]
@WeidiXie Have you reproduced the number reported in paper?

**Comment by fyangneil** [comment]
> Do you get the same results as in here http://rpg.ifi.uzh.ch/data/detection_results.zip ? Did you use this [evaluation script](https://github.com/antonilo/unsupervised_detection/blob/master/test_generator.py)? If not, I might need more details to debug the problem (TensorFlow version used, for example)
I used the script. the tensorflow version is 13.1.

**Comment by WeidiXie** [comment]
I see, thanks, I was just wondering the step-wise breakdown results, but I'll test this myself when I got time.
